### PR TITLE
Remove laika submodule

### PR DESCRIPTION
**Pull request by fredyshox**

### CODE REVIEW

**Comment by fredyshox** [comment]
Blocked by https://github.com/commaai/openpilot/pull/30452

File: SConstruct [hunk 1]
```diff
@@ -56,7 +56,7 @@ AddOption('--pc-thneed',
 AddOption('--minimal',
           action='store_false',
           dest='extras',
-          default=os.path.islink(Dir('#laika/').abspath),
+          default=True,
```
Comment (fredyshox) [inline SConstruct:59]: @adeebshihadeh what was this check for?

**Comment by haraschax** [comment]
You'll need to make xx PR with passing tests with the op version before we can merge

File: SConstruct (hunk 1 above)
Comment (adeebshihadeh) [reply SConstruct:59]: to check if we're on a master or release branch, you can change to rednose

**Pull request merged**